sphinx_gallery/gen_gallery.py

```diff
@@ -86,6 +86,7 @@ def generate_gallery_rst(app):
         abort_on_example_error=app.builder.config.abort_on_example_error)
 
     # this assures I can call the config in other places
```

Comment:
This comment is related to the line `app.config.sphinx_gallery_conf = gallery_conf`, isn't it?